### PR TITLE
fix: cosmwasm 1.0

### PR DESCRIPTION
**Pull request by nowooj**

### CODE REVIEW

**Review by caramis — APPROVED** [review]
LGTM

File: content/en/docs/develop/guides/smart-contract-guide/interacting-with-the-contract.md [hunk 1]
```diff
@@ -137,11 +140,14 @@ Check out your contract information:
...
 ```sh
 xplad query wasm contract xpla18vd8fpwxzck93qlwghaj6arh4p7c5n896xzem5
-address: xpla18vd8fpwxzck93qlwghaj6arh4p7c5n896xzem5
-owner: xpla1dcegyrekltswvyy0xy69ydgxn9x8x32zdtapd8
-codeid: 1
-initmsg: eyJjb3VudCI6MH0=
-migratable: false
+contract_info:
+  admin: xpla1dcegyrekltswvyy0xy69ydgxn9x8x32zdtapd8
+  codeid: "1"
```
Comment (jhlee-young) [inline content/en/docs/develop/guides/smart-contract-guide/interacting-with-the-contract.md]: codeid -> code_id

File: content/en/docs/develop/guides/smart-contract-guide/interacting-with-the-contract.md [hunk 2]
```diff
@@ -137,11 +140,14 @@ Check out your contract information:
 
 ```sh
 xplad query wasm contract xpla18vd8fpwxzck93qlwghaj6arh4p7c5n896xzem5
-address: xpla18vd8fpwxzck93qlwghaj6arh4p7c5n896xzem5
-owner: xpla1dcegyrekltswvyy0xy69ydgxn9x8x32zdtapd8
-codeid: 1
-initmsg: eyJjb3VudCI6MH0=
-migratable: false
+contract_info:
```
Comment (jhlee-young) [inline content/en/docs/develop/guides/smart-contract-guide/interacting-with-the-contract.md:310]: plz leave the `address` line

Comment (nowooj) [reply content/en/docs/develop/guides/smart-contract-guide/interacting-with-the-contract.md:310]: https://github.com/xpladev/docs/pull/6/commits/99d0194f4f8faa7fa2b55fed45007f1e5fcbc9b0

File: content/en/docs/develop/guides/smart-contract-guide/interacting-with-the-contract.md (hunk 1 above)
Comment (nowooj) [reply content/en/docs/develop/guides/smart-contract-guide/interacting-with-the-contract.md]: https://github.com/xpladev/docs/pull/6/commits/99d0194f4f8faa7fa2b55fed45007f1e5fcbc9b0

**Review by jhlee-young — APPROVED** [review]
LGTM

**Pull request merged**